src/styles/blocks/section2.scss

```diff
@@ -0,0 +1,67 @@
+.section_2 {
+  margin-top: 79px;
+  text-align: justify;
+  width: 166px;
+  height: 81px;
+}
+
+.section_2__stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 5px;
+}
+
+.section_2__stars_star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin: 0;
+  background-position: center;
+}
+
+.section_2__stars_star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.section_2__stars_text {
+  text-align: right;
+  height: 16px;
+  width: 53px;
+  line-height: 16px;
+  font-size: 10px;
+  margin-left: 10px;
+  margin-bottom: 0;
+  margin-top: 0;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.section_2_price {
+  display: flex;
+  height: 18px;
+  width: 166px;
+  justify-content: space-between;
+  margin: 22px 0 17px;
```

Comment:
The margins here are used to create spacing between elements. Consider using padding on the parent element or margin on child elements for consistency and to adhere to the checklist's code style rules.